src/plugins/cosmos/components/modals/Staking/views/ClaimConfirm.tsx

```diff
@@ -34,16 +41,12 @@ export type StakingValues = {
 
 type ClaimConfirmProps = {
   assetId: CAIP19
-  cryptoStakeAmount: BigNumber
-  fiatAmountAvailable: string
+  accountSpecifier: CAIP10
 }
 
-// TODO: Wire up the whole component with staked data
-export const ClaimConfirm = ({
-  assetId,
-  cryptoStakeAmount,
-  fiatAmountAvailable
-}: ClaimConfirmProps) => {
+const SHAPESHIFT_VALIDATOR_ADDRESS = 'cosmosvaloper199mlc7fr6ll5t54w7tts7f4s0cvnqgc59nmuxf'
```

Comment:
Nitpick for sure, but wonder if this should live in `lib`?

Reply:
You can expect it will go away either before, or right after release, I'm still on the hunt for all hardcoded things and the ShapeShift validator address can be surfaced from the validator slice in https://github.com/shapeshift/web/pull/1356